.gitignore

```diff
@@ -6,12 +6,11 @@
 /local-config-extensions.php
 /sql-dump-*.sql
 /db-sync
-/content
 .DS_Store
 .idea
 config.local.yaml
 /.wp-cli
 /wp-cli.local.yml
-/extensions
```

Comment:
This directory is actually being tracked as [it contains the `example` extension](https://github.com/Chassis/Chassis/tree/c187e6e5c9be9f66322b683025a66af541181f58/extensions/example).

Reply:
This is intentional; `gitignore` simply ensures untracked files are not shown, but still allows files to be committed within it. Chassis requires the `example` extension for technical reasons (plus it's useful), but we don't want to show any other files in that directory.

Reply:
@rmccue Right, so we want to keep tracking `/extensions` directory and the `example` directory inside it while ignoring everything else. This is why I suggest we introduce a `.gitignore` in the `extensions` directory` that only tracks the `example` directory and excludes everything else.

Currently it is a bit confusing to see the `extensions` directory ignored in the `.gitignore` but actually tracked in the repository. I think it only works because it was once tracked and later ignored in 9b49b5637a6b614a4d16ff1320aed80f38aa6105. Per [docs](https://git-scm.com/docs/gitignore):

> Files already tracked by Git are not affected.

and

> To stop tracking a file that is currently tracked, use `git rm --cached`.

So if the intention is to track the sample extension `extensions/sample`, I suggest we clarify that by updating the `.gitignore` files.

I'm totally OK with not changing this, if you prefer. Just wanted to explain the reasoning and though behind it.